ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -117,6 +117,7 @@ ZTM:
   is_abstract: true
   opt_uses:
   - discharge_air_temperature_sensor
+  - zone_air_relative_humidity_sensor
```

Comment:
there is a type called `ZHM` that already defines this. please use it explicitly.